examples/dataflow-custom-templates/go/go.sum

```diff
@@ -0,0 +1,668 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+cloud.google.com/go v0.34.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
We can exclude the checksum file in order to keep the size of overall repo small.

Reply:
@prathapreddy123 This is an autogenerated file that is version controlled.  See https://github.com/apache/beam/blob/master/sdks/go.sum for example.

Reply:
Additional examples:
1. https://github.com/GoogleCloudPlatform/cloud-sql-proxy/blob/main/go.sum
2. https://github.com/GoogleCloudPlatform/terraform-validator/blob/main/go.sum
3. https://github.com/uber-go/zap/blob/master/go.sum